src/main/java/seedu/duke/data/TaskList.java

```diff
@@ -47,6 +47,13 @@ public Task removeTask(int index) throws NoSuchTaskException {
         return task;
     }
 
+    /**
+     * Adds tag to the task list.
+     *
+     * @param tagDescription The description of tag that is inputted by user
+     * @param index The task number to be added with tag
```

Comment:
Index isn't really the same as task number. Maybe change the description to `Index of task to be added with tag.`